waspc/data/Generator/templates/server/src/dbClient.ts

```diff
@@ -1,17 +1,10 @@
 {{={= =}=}}
 import Prisma from '@prisma/client'
 
-{=# isAuthEnabled =}
-import { registerAuthMiddleware } from './core/auth/prismaMiddleware.js'
-{=/ isAuthEnabled =}
 
 const createDbClient = () => {
   const prismaClient = new Prisma.PrismaClient()
 
-  {=# isAuthEnabled =}
-  registerAuthMiddleware(prismaClient)
```

Comment:
We no longer have the password hashing middleware 😢

Reply:
Ok! I should probaly know after reviewing this pr, but why is that so?

Reply:
Before we had the `User` entity, but now we have `User -> Auth -> AuthIdentity` chain that gets saved when we create or update the `providerData`. Before it was easy to act on `User` saves and only on the `password` field.

`providerData` is a `String` in the DB which we need to serialize and deserialize on saves. But ... we can't make a middleware that is okay with us giving an object to Prisma _and then in the middleware_ serialising it. 

I'll test this one more time in a separate Prisma project just to confirm 100% we can't do that and then I'll report back here.

Reply:
Ok, no prob, but when does password hashing occur then?

Reply:
So wat happens how if they want to directly do something with `User` model in Prisma? If they want to save password to it, they need to make sure they hash it, right?

I saw some other solutions using `hashedPassword` for the field name, should we do the same? I like the idea.

Reply:
Developers doing stuff with `AuthIdentity` need to take care of password hashing by hand. No more "it's handled by us" guarantee.

We are hashing it when serialising the `providerData`.

We can rename the field to `hashedPassword`, sure 👍

Reply:
Ok, got it. Interesting idea: what if we had a Prisma middleware (or whatever is this functionality now called) that would not hash password, but would check that it is hashed? I am not sure how easy it is to implement such check, but in theory we could I think -> we could check it has certain specific length, or we could add some prefix to our hashed passwords, something like "hashed:", and if they don't have that, we complain.

You will probably feel this is too much for now, in which case I understand, but wanted to raise the idea of it for the future. Feel free to ignore it for now (or maybe create an issue for it, or maybe not).

Renaming to `hashedPassword` -> yeah, let's do that.

Reply:
> Ok, got it. Interesting idea: what if we had a Prisma middleware (or whatever is this functionality now called) that would not hash password, but would check that it is hashed? I am not sure how easy it is to implement such check, but in theory we could I think -> we could check it has certain specific length, or we could add some prefix to our hashed passwords, something like "hashed:", and if they don't have that, we complain.

I'm a bit worried about logic based on the `hashedPassword` value. What if the hash length is not consistent? Or somebody has a password that's the same length as our hash? Or somebodies password starts with `hashed:` (It's a stretch, but it's not 0% chance)

I've renamed the field 🚀